GroundDBMemoryPool/src/rdma/rdma_client/connect.cc

```diff
@@ -0,0 +1,40 @@
+#include <rdma.hh>
```

Comment:
`connect` basically is a wrap up of `connect` in [https://github.com/alvinkwok1/RDMA-EXAMPLE/](url)